fastapi_sqlalchemy/applications.py

```diff
@@ -1,124 +1,36 @@
 """ The main application """
-from typing import Any, Dict, List, Optional, Union
-
-from sqlalchemy.engine import Engine, Connection, create_engine
-
+from typing import Optional, Union
 
 from fastapi import FastAPI
-from fastapi.openapi.docs import (
-    get_redoc_html,
-    get_swagger_ui_html,
-    get_swagger_ui_oauth2_redirect_html,
-)
-from fastapi.exception_handlers import (
-    http_exception_handler,
-    request_validation_exception_handler,
-)
-from fastapi.exceptions import RequestValidationError
-
-from starlette.requests import Request
-from starlette.responses import HTMLResponse, JSONResponse
-from starlette.exceptions import HTTPException
-from starlette.middleware.base import BaseHTTPMiddleware
-from starlette.routing import BaseRoute
+from sqlalchemy.engine import create_engine, Engine
+from sqlalchemy.orm import sessionmaker
 
 from fastapi_sqlalchemy.models import Session, BASE
-from fastapi_sqlalchemy.middleware import session_middleware
 
 
 class FastAPI_SQLAlchemy(FastAPI):
     """ Application class """
 
-    def __init__(
-            self,
-            bind: Union[str, Engine, Connection, None],
-            debug: bool = False,
-            routes: List[BaseRoute] = None,
-            template_directory: str = None,
-            title: str = "Fast API",
-            description: str = "",
-            version: str = "0.2.0",
-            openapi_url: Optional[str] = "/openapi.json",
-            openapi_prefix: str = "",
-            docs_url: Optional[str] = "/docs",
-            redoc_url: Optional[str] = "/redoc",
-            **extra: Dict[str, Any],
-    ):
-        super().__init__(
-            debug=debug,
-            routes=routes,
-            template_directory=template_directory,
-            title=title,
-            description=description,
-            version=version,
-            openapi_url=openapi_url,
-            openapi_prefix=openapi_prefix,
-            docs_url=docs_url,
-            redoc_url=redoc_url,
-            **extra
-        )
-        if bind is not None:
-            if isinstance(bind, str):
-                self.bind = create_engine(bind, pool_pre_ping=True)
-            else:
-                self.bind = bind
-            Session.configure(bind=self.bind)
+    bind: Optional[Engine] = None
```

Comment:
Remove (and it could be something other than an Engine, e.g. Connection)

Reply:
Done